doc/nrf/installation/install_ncs.rst

```diff
@@ -0,0 +1,617 @@
+.. _install_ncs:
+.. _auto_installation:
+.. _gs_installing:
+
+Installing the |NCS|
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+There are different ways to install the |NCS|, depending on your preferred development environment and the :ref:`toolchain management tool <toolchain_management_tools>`:
+
+* Using |VSC| and the |nRFVSC| (recommended)
+* Using command line and nRF Util
+
+Regardless of which way you choose, the following steps install the |NCS| source code and the |NCS| :term:`toolchain`.
+This includes everything that is required by Zephyr's :ref:`zephyr:getting_started` together with additional tools and Python dependencies that the |NCS| uses.
+
+.. rst-class:: numbered-step
+
+Update operating system
+***********************
+
+Before you start setting up the toolchain, install available updates for your operating system.
+See :ref:`requirements` for information on the supported operating systems.
+
+.. _installing_vsc:
+
+.. rst-class:: numbered-step
+
+Install prerequisites
+*********************
+
+Depending on your preferred development environment, install the following tools:
+
+.. tabs::
+
+   .. group-tab:: nRF Connect for Visual Studio Code
+
+      * The latest version of |VSC| for your operating system from the `Visual Studio Code download page`_.
+      * In |VSC|, the latest version of the |nRFVSC|.
+        For installation instructions, see `How to install the extension`_.
```

Comment:
This is weird because it links to https://nrfconnect.github.io/vscode-nrf-connect/get_started/install.html which in turn then links to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/installation/assistant.html. Something is not right, I think we need @trond-snekvik  here.

Reply:
Let me simplify this.

Reply:
Pointed directly to VSC Marketplace. The page in the VSC docs will be rewritten once this PR is merged.